lms/djangoapps/grades/signals/handlers.py

```diff
@@ -133,6 +133,14 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
             kwargs.get('points_possible'),
         )
     )
+    log.info(
+        u'Task: {}. Begin async calculation of subsection grades for {} in {} / {}'.format(
+            result.id,
+            kwargs['user_id'],
+            kwargs['course_id'],
+            kwargs['usage_id'],
+        )
+    )
```

Comment:
So we have a few prefixes in use: 
* In grades/new/subsection_grade.py: `Persistent Grades:`
* In grades/transformers.py: `GradesTransformer:`
* Elsewhere in grades/signals/handlers.py: `Grades:`.

Since This is coming out of the signal handler, I'm copying the nearest convention, and prefixing with `Grades:`, but I'm happy to put in a little work to standardize our logging a little more, if that's desirable.